appveyor.yml

```diff
@@ -26,6 +26,15 @@
     - cmd: >-
         subst s: c:\projects
     - ps: set-location -path s:\node_modules\node-sass
+    - ps: Uninstall-Product node 4 $env:platform
+    - ps: Write-Host "Installing node-chakracore..." -ForegroundColor Cyan
+    - ps: Write-Host "Downloading..."
+    - ps: $msiPath = "$env:USERPROFILE\node-chakracore-$env:platform.msi"
+    - ps: $file = "https://github.com/nodejs/node-chakracore/releases/download/node-chakracore-7.0.0-pre9/node-chakracore-$($env:platform).msi"
```

Comment:
@kunalspathak no. IMO this is ready to go it's just a matter of finding the appropriate time to release it. I suspect it will after LibSass 3.4 stable see #1800.